### DISCUSSION TITLE
createPopper

### DISCUSSION THREAD

**Discussion by pavankumarAN** [twbs/bootstrap#32990] [Q&A]
I'm developing an application by referring to tutorials on Udemy.com along with angular. I have installed bootstrap 5 by using npm i bootstrap@next as per instruction in the documentation. When I tried to create a button group I'm getting an error as "Uncaught TypeError: Cannot read property 'createPopper' of undefined".
I have inserted some of the configurations into angular.json like

 "styles": [
              "node_modules/bootstrap/dist/css/bootstrap.min.css",
              "src/styles.css"
            ],
            "scripts": [
              "node_modules/bootstrap/dist/js/bootstrap.min.js"
            ]
<img width="1276" alt="Screenshot 2021-02-05 at 12 56 56 PM" src="https://user-images.githubusercontent.com/35585229/107002726-ad578b00-67b1-11eb-8f63-2c47cc4928d4.png">

The code which I used in my component is pasted below
`<div class="btn-group" role="group">
            <button id="btnGroupDrop1" type="button" class="btn btn-primary dropdown-toggle" data-bs-toggle="dropdown" aria-expanded="false">
              Dropdown
            </button>
            <ul class="dropdown-menu" aria-labelledby="btnGroupDrop1">
              <li><a class="dropdown-item" href="#">Dropdown link</a></li>
              <li><a class="dropdown-item" href="#">Dropdown link</a></li>
            </ul>
          </div>`
          
     NOTE: Other than running the installation command of bootstrap I didn't run for anything like jQuery and other stuff. I'm new to bootstrap and I'm new to reporting issues also on github. This is my first report. Help me to grow.

**Comment by ManarKaram** [comment]
Make sure that you're using the popper.js or use the bundle.js file.
and make sure that you're using the latest version of bootstrap 5.0.0 beta2.
I tried to use the cdn of bootstrap bundle.js and it worked then I redownloaded the files again so it worked fine.

**Reply by omarahmed8k** [reply]
Thank you Manar
